src/NuGet.Clients/NuGet.PackageManagement.UI/Models/Package/ReferencedPackageModel.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using NuGet.Packaging;
+using NuGet.Packaging.Core;
+
+namespace NuGet.PackageManagement.UI.Models
+{
+    public class ReferencedPackageModel : LocalPackageModel
```

Comment:
We talked about this offline. ReferencedPackageModel is not a type of LocalPackageModel as this inheritance hierarchy suggests. We will consider having both these package types share a base class or local capability of sorts. We can handle that in a future PR.